app/assets/javascripts/oxalis/view/right-menu/dataset_info_tab_view.js

```diff
@@ -226,14 +257,31 @@ class DatasetInfoTabView extends React.PureComponent<DatasetInfoTabProps> {
       Store.getState().temporaryConfiguration.controlMode === ControlModeEnum.VIEW;
 
     const zoomLevel = calculateZoomLevel(this.props.flycam, this.props.dataset);
-
+    const extentInVoxel = getDatasetExtentInVoxel(this.props.dataset);
+    const extent = getDatasetExtentInLength(this.props.dataset);
+    const datsetExtents = [
+      `${extentInVoxel.width} x ${extentInVoxel.height} x ${extentInVoxel.depth}`,
+      formatExtentWithLength(extent),
+    ];
     return (
       <div className="flex-overflow info-tab-content">
         {this.getTracingName(isPublicViewMode)}
         {this.getDatasetName(isPublicViewMode)}
 
-        <p>Viewport Width: {formatZoomLevel(zoomLevel)}</p>
+        <p>Viewport Width: {formatNumberToLength(zoomLevel)}</p>
         <p>Dataset Resolution: {formatScale(this.props.dataset.dataSource.scale)}</p>
+        <p>
+          <table>
+            <tr>
+              <td style={{ paddingRight: 8 }}>Dataset Extent:</td>
+              <td>{datsetExtents[0]}</td>
+            </tr>
+            <tr>
+              <td />
```

Comment:
Do you think it is clear from a UI perspective that this row displays the extent in voxels? Maybe it should read that somewhere (either in the first cell of the row) or "5 x 10 x 15 Voxel" or (more mathematically correct) "5 Voxel x 10 Voxel x 15 Voxel" or "5 x 10 x 15 Voxel³". What do you think, @MichaelBuessemeyer  @daniel-wer ?